registry/uiLocaleKeysBackend.json

```diff
@@ -368,6 +368,7 @@
   "submission.contributors",
   "submission.files",
   "submission.list.assignEditor",
+  "submission.list.changeSubmissionLanguage.title",
```

Comment:
I think you can ignore the changes in this file -- Jarda is managing it extra -- but let me double check with Jarda...